content/en/docs/collector/configuration.md

```diff
@@ -303,7 +303,7 @@ exporters:
   jaeger:
     endpoint: "jaeger-all-in-one:14250"
     tls:
-      insecure: true
+      insecure: false
```

Comment:
Isn't `false` the default value?  If we're going to set this it should probably be removed instead.  That said, does this configuration actually work without the user taking steps to ensure a valid TLS cert on the other end?  Are those steps documented here?

Reply:
I was just keeping it to illustrate that it's an option that can be configured, while avoiding an insecure example

Reply:
The example as it is will fail, as it's not specifying which CA to use when doing validation of the server's certificate.

Reply:
So is the vote to close this PR @jpkrohling @Aneurysm9? Or request further changes?

Reply:
We should do one of those two things.  I'm not sure that I care overmuch which it is.

Reply:
I'm happy to add to the docs and specify an example CA bundle

Reply:
It looks like specifying a CA bundle is optional, but a cert and key is required for mTLS according to [this](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/jaegerexporter#getting-started) and [this](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#tls--mtls-configuration).

Reply:
Specifying a CA bundle is optional, as the target server might have a cert signed by a known root CA. This is not the case for self-signed certs. I know this would make the guide longer, but I do think we should instruct people how to generate the certs for dev/testing purposes (hinting that they might want to find an appropriate way to generate/provision the production certs), and use those certs in the examples.

Otherwise, people will just copy/paste the insecure config into their production's config.

Reply:
If you need, use the steps I outlined in this blog post to generate the certs: https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f

Reply:
@jpkrohling what do you think now?  I should have aligned it to your blog post, with some minor tweaks.

Reply:
I'd vote like @Aneurysm9 and remove `insecure: false` if it is the default.